src/wordpress/config.py

```diff
@@ -90,9 +128,18 @@ def is_install_valid(self):
 
     @property
     def wp_version(self):
+        """
+        Returns installed WordPress version
+        """
         return self.run_wp_cli('core version')
 
     def config_infos(self, field=None):
+        """
+        Extract and return WordPress configuration informations
```

Comment:
i.e the ones from `wp-cli config get` (https://developer.wordpress.org/cli/commands/config/get/)

Reply:
fertig